hack/codegen.sh

```diff
@@ -139,23 +139,7 @@ statik-gen elevator config/internal/elevator
 # API server
 # -------
 # RBAC
-$CONTROLLER_GEN rbac:roleName=manager paths="./pkg/api-server/..." output:rbac:artifacts:config=config/internal/api-server/rbac
-# The `|| true` is because the `,s/ClusterRole/Role/g` will error out if there is no match of `ClusterRole` (eg., the file is empty) in the file.
-ed config/internal/api-server/rbac/role.yaml <<EOF || true
-,s/ClusterRole/Role/g
-w
-EOF
-statik-gen api-server config/internal/api-server
-
-# API server
-# -------
-# RBAC
-$CONTROLLER_GEN rbac:roleName=manager paths="./pkg/api-server/..." output:rbac:artifacts:config=config/internal/api-server/rbac
-# The `|| true` is because the `,s/ClusterRole/Role/g` will error out if there is no match of `ClusterRole` (eg., the file is empty) in the file.
-ed config/internal/api-server/rbac/role.yaml <<EOF || true
-,s/ClusterRole/Role/g
-w
-EOF
+$CONTROLLER_GEN rbac:roleName=manager-role paths="./pkg/api/v1/services/..." output:rbac:artifacts:config=config/internal/api-server/rbac
```

Comment:
This is giving too much permission in a cluster role...i.e. secrets should only be given in the same namespace as the APIServer deployment for TLS purposes, most definitely not cluster-wide.